### PR TITLE
Site Manger Dashboard: Participant Details

### PR DESCRIPTION
**Pull request by abhinavjonnada82**
This PR addresses the following features:
-> Displays Participant details on a table
-> Lets the user Update fields on a modal
-> Provides admin audit log
-> HTTP POST request to UpdateParticpant API
-> Checked for GIT conflicts as well

Test Cases Verified: 
- [x] When you make a change on the UI. The results are updated on the UI and on the backend as well
- [x] On the Edit modal, shows the field which is going to get modified which is also an input field 
- [x] Added some error fallback cases 
- [x] Once your done you would get a time stamp 
- [x] You can cancel changes if they’re not saved

POC: Available on request.
Note: Placeholders used for USER_ID Field & concept_id for alternated contact details

### CODE REVIEW

File: siteManagerDashboard/participantDetails.js [hunk 1]
```diff
@@ -1,33 +1,516 @@
...
+                                        <button type="button" id="cancelChanges" class="btn btn-danger">Cancel Changes</button>
+                                        &nbsp;
+                                        <b>Last Modified by: <span id="modifiedId"></span></b>
+                                    </div>
+                            </div>
+                        </div>
+        `;
+
+
+        template += ` <div class="modal fade" id="modalShowMoreData" data-keyboard="false" data-backdrop="static" tabindex="-1" role="dialog" data-backdrop="static" aria-hidden="true">
```
Comment (bhaumik55231) [inline siteManagerDashboard/participantDetails.js:168]: Can we just have 1 modal and reuse it for different purposes?

File: siteManagerDashboard/utils.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+
+export function getCurrentTimeStamp() {
```
Comment (bhaumik55231) [inline siteManagerDashboard/utils.js:3]: To caputre timestamp we can directly use ```new Date().toISOString()```

**Review by bhaumik55231 — COMMENTED** [review]
I have added some comments.

File: siteManagerDashboard/participantDetails.js (hunk 1 above)
Comment (abhinavjonnada82) [reply siteManagerDashboard/participantDetails.js:168]: Sure1

File: siteManagerDashboard/utils.js (hunk 2 above)
Comment (abhinavjonnada82) [reply siteManagerDashboard/utils.js:3]: Ahhh! dint know abt that.

**Review by bhaumik55231 — APPROVED** [review]

File: siteManagerDashboard/participantDetails.js [hunk 3]
```diff
@@ -1,33 +1,496 @@
...
+                                        <th scope="col">Relationship</th>
+                                        <th scope="col">Home Number</th>
+                                        <th scope="col">Mobile Number</th>
+                                        <th scope="col">Email</th>
+                                        <th scope="col"></th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        <tr class="detaileAltRow">
+                                        <td>${participant.Module1 && participant.Module1.ALTCONTACT1 !== undefined ? participant.Module1.ALTCONTACT1.ALTCONTACT1_FNAME : ""} `+` ${participant.Module1  && participant.Module1.ALTCONTACT1 !== undefined ? participant.Module1.ALTCONTACT1.ALTCONTACT1_LNAME : ""} </td>
```
[the remark this answers is not in the document]
Comment (abhinavjonnada82) [reply siteManagerDashboard/participantDetails.js:145]: undefined removed

**Review by hzhao392 — APPROVED** [review]

**Pull request merged**